kit/inputs/src/UIMaterialTextView/hooks/useApplyMask/amount/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const UIConstants = {
+    aspectRatio: {
```

Comment:
Maybe to pick a better naming? Aspect ratio usually refers to sizes

Reply:
Maybe `accuracy`?

Reply:
Changed to `decimalAspect` as commonly used in UIKit.